includes/Modules/Analytics.php

```diff
@@ -277,17 +277,17 @@ protected function handle_provisioning_callback() {
 		}
 
 		// Check for a returned error.
-		$error = $input->filter( INPUT_GET, 'error', FILTER_SANITIZE_STRING );
+		$error = htmlspecialchars( $input->filter( INPUT_GET, 'error' ) );
```

Comment:
@kuasha420 I think `error` might be `null` which in that case PHP will complain about `passing null to parameter #1 ($string) of type string is deprecated `